main/src/ca/uwaterloo/flix/language/errors/SafetyError.scala

```diff
@@ -116,4 +117,132 @@ object SafetyError {
     })
   }
 
```

Comment:
I will wait to comment on the errors since you will probably change them a lot :)